site/docs/v1/tech/messengers/generic-messenger.adoc

```diff
@@ -80,3 +80,43 @@ In these fields, you can add custom key and value pairs to augment your headers.
 image::messengers/configuration-test-generic.png[Test Configuration Generic, role=top-cropped, width=1200]
 
 You also can test your generic messenger configuration. By hitting `Test generic configuration` FusionAuth will fire a JSON message to the messenger just created to ensure everything is set up correctly.
+
+== Writing the Generic Message Receiver
+
+You need to write a server side component, a receiver, which will be deployed at a URL and send the actual messages. This component can be written in any language that can consume a JSON message.
```

Comment:
> deployed at a URL and send the actual messages. 

This almost sounds like the "receiver" is sending the messages. What if we said something like:

> deployed at a URL to consume the message sent from FusionAuth, and proxy the request to the 3rd party messaging service.